app/helpers/application_helper.rb

```diff
@@ -1,4 +1,10 @@
 module ApplicationHelper
+  # This is a path helper that routes the main search
+  # form based on the current search context.
+  def searchworks_search_action_path(opts = {})
+    return article_home_path(opts) if article_search?
```

Comment:
I think this will need to change to `article_index_path` when that route becomes available.